polling_stations/apps/file_uploads/views.py

```diff
@@ -247,13 +247,16 @@ def get_context_data(self, **kwargs):
             .values_list("polling_station_id", "uprn")
         )
 
-        for station in council_from_default_db.pollingstation_set.all():
+        for station in council_from_default_db.pollingstation_set.all().annotate(
+            pk=F("id")
+        ):
```

Comment:
Can you explain what this is doing? `pk` should already be set as a property on the object, or if it's not meant to be the built in `pk` then that name might b confusing

Reply:
I'm not sure!  
```python
        for station in council_from_default_db.pollingstation_set.all():
            context["STATIONS"].append(
                {
                    "address": station.address,
                    "postcode": station.postcode,
                    "location": "✔️" if station.location else "❌",
                    "example_uprn": example_uprn_map.get(station.internal_council_id),
                    "pk": station.id,
                }
            )
```
works fine!